pkg/ceph/client/crush.go

```diff
@@ -69,14 +70,27 @@ rule replicated_ruleset {
 # end crush map
 `
 
-func GetCrushMap(context *clusterd.Context, clusterName string) (string, error) {
+type CrushMap struct {
```

Comment:
no buckets in this model of a crush map?

Reply:
i didn't need them for my purposes, but i'll go ahead and fill this struct in more